editor/import/resource_importer_obj.cpp

```diff
@@ -411,6 +411,19 @@ static Error _parse_obj(const String &p_path, List<Ref<Mesh>> &r_meshes, bool p_
 
 				mesh = surf_tool->commit(mesh, mesh_flags);
 
+				if (mesh_flags & RS::ARRAY_FLAG_COMPRESS_ATTRIBUTES && generate_tangents) {
+					// Compression is enabled, so let's validate that the normals and tangents are correct.
+					Vector<Vector3> norms = array[Mesh::ARRAY_NORMAL];
```

Comment:
These changes depend on:
* https://github.com/godotengine/godot/pull/86365

It seems

Reply:
Right, that's something I wanted to check with @clayjohn on Monday.

Reply:
I included that PR for now as it seemed harmless enough.